atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,139 @@
+        # Compute pairwise cosine similarities between medoids
+        similarity_matrix = cosine_similarity(medoids)
+
+        # TODO: Delete all of the clusters here
+        # Expose clusters and similarity matrix as instance variables
+        for index in range(len(medoids)):
+            cluster = { "properties": [{
+                "cluster_id": "",
+                "name": str(index),
+                "members": embeddings_uuids[labels == index].tolist(),
+                }]}
+            self.weaviate_client.add_embeddings(CollectionNames.CLUSTERCENTER.value, medoids[index].tolist(), cluster)
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)